app/src/forms/auth/service.js

```diff
@@ -201,19 +201,11 @@ const service = {
     };
   },
 
-  login: async (token, params = {}, loadForms = true) => {
+  login: async (token) => {
     const userInfo = service.parseToken(token);
     const user = await service.getUserId(userInfo);
 
-    if (!loadForms) {
-      return { ...user };
-    }
-
-    const forms = await service.getUserForms(user, params); // get forms for user (filtered by params)...
-    params.active = false;
-    const deletedForms = await service.getUserForms(user, params); // get forms for user (filtered by params)...
-
-    return { ...user, forms: forms, deletedForms: deletedForms };
+    return { ...user };
```

Comment:
I wasn't sure about this duplication - we were duplicating the `user` object before to include it in the result. I don't think we make any downstream changes to the object that would cause problems with the original? We probably don't need the copy now? I think this could just be:

```suggestion
    return user;
```